docs/_data/mesheryctlcommands/lifecycle/system/configure/command/config.yaml

```diff
@@ -1,4 +1,4 @@
 name: config
 description: Configures Meshery to use a Kubernetes cluster.
 usage: |
-    mesheryctl system config [minikube | gke | aks | eks ] [flags] [flags]
+    mesheryctl system config [ minikube | gke | aks | eks ] [flags] [flags]
```

Comment:
Actually, a convention of no whitespace adjacent to the square bracket is fine and follows our current convention.

Are [flags] accidentally included twice?